functions/__fisher_path_update.fish

```diff
@@ -5,7 +5,7 @@ function __fisher_path_update -a path
         git rebase --abort
         git fetch --quiet origin master
         git reset --quiet --hard FETCH_HEAD
-        git clean --quiet -d --force
+        git clean --quiet -dx --force
```

Comment:
So, without this `x` untracked files will remain in the repo?

What about `-X`?